examples/DhtTemperatureAndHumiditySensor/DhtTemperatureAndHumiditySensor.ino

```diff
@@ -62,95 +74,218 @@ static const uint8_t FORCE_UPDATE_N_READS = 10;
 
 #define CHILD_ID_HUM 0
 #define CHILD_ID_TEMP 1
+#define CHILD_ID_TEMP_HI 2
+
+// Set this offset if the sensors have permanent small offsets to the real temperatures/humidity.
+// In Celsius degrees or moisture percent
+#define SENSOR_HUM_OFFSET 0       // used for temperature data and heat index computation
+#define SENSOR_TEMP_OFFSET 0      // used for humidity data
+#define SENSOR_TEMP_HI_OFFSET 0   // used for heat index data
 
+
+
+#include <MySensors.h>  
+#include <Adafruit_Sensor.h>
+#include <DHT_U.h>
+
+DHT_Unified dhtu(DHTDATAPIN, DHTTYPE);
+// See guide for details on Adafruit sensor wiring and usage:
+//   https://learn.adafruit.com/dht/overview
+
+uint32_t delayMS;
 float lastTemp;
 float lastHum;
-uint8_t nNoUpdatesTemp;
-uint8_t nNoUpdatesHum;
+uint8_t nNoUpdates = FORCE_UPDATE_N_READS; // send data on start-up 
 bool metric = true;
+float temperature;
+float humidity;
+float temperatureHI;
+
+
 
 MyMessage msgHum(CHILD_ID_HUM, V_HUM);
 MyMessage msgTemp(CHILD_ID_TEMP, V_TEMP);
-DHT dht;
+MyMessage msgTempHI(CHILD_ID_TEMP_HI, V_TEMP);
```

Comment:
a better name would be "msgHeatIndex", CHILD_ID_HEATINDEX

Reply:
I appreciate your suggestion.
I'm going to change it